charts/vela-core/templates/cluster-gateway.yaml

```diff
@@ -4,6 +4,8 @@ kind: Deployment
 metadata:
   name: {{ .Release.Name }}-cluster-gateway
   namespace: {{ .Release.Namespace }}
+  annotations:
+    helm.sh/resource-policy: keep
```

Comment:
Why do we choose to keep this resource after vela-core is uninstalled? https://helm.sh/docs/howto/charts_tips_and_tricks/#tell-helm-not-to-uninstall-a-resource

Reply:
the other way round, the idea is to make the chart installation compatible w/ existing cluster-gateway instances installed via OCM. anyway, i stripped the chart changes from this thread.